src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -420,6 +420,10 @@ public Command handleCubeIntakeButton(){
     }
   }
 
+  public ArmState getArmState() {
+    return currentState;
```

Comment:
currentState is actually the desired state, so it won't be there yet as soon as the state changes. So currently using this will result in dropping the cone early.

Reply:
From what I understand, even if we stop running the command early, the intake still holds the piece. What we're working on now is preventing that piece from popping out from contact with other robots/field elements. 

We implement stopping intake here similarly to how it's done in auton (ie. we stop intake before we go into the drop sequence) so it should hopefully be fine, but I could be wrong.

Reply:
Turns out that arm movement while realigning can cause game pieces to drop. The way drivers currently do it is to hold down until the arm goes into the drop state, so I've changed the code to reflect that.

Supposedly the new arm design won't move as much while aligning, but that remains to be seen